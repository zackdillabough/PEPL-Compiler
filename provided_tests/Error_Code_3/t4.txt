POLY A = x + 1;
POLY B = x^2 + x + 4;
POLY C(X) = X - 1;
START
INPUT X;
D(X);
G2(0);
F(1, G2(0));
D(E(A(F(B(F(F(F3(F4(F3(F4(F3(F4(F3
(F4(F3(F4(S(F(F3(X))))))))))))))))))));
363 2 2 1 0
