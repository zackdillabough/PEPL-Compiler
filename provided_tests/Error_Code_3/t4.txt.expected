Error Code 3: 6 7 8 8 9 9 9 9 9 9 9 9 9 9 9 9 10 10 10 10 10 10 